iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings

```diff
@@ -2031,15 +2037,15 @@
 /* These translations are used for all type.highway.*.bridge. */
 "type.highway.footway.bridge" = "Puente";
 
-"type.highway.footway.demanding_alpine_hiking" = "Camino";
+"type.highway.footway.demanding_alpine_hiking" = "Path";
 
-"type.highway.footway.demanding_mountain_hiking" = "Camino";
+"type.highway.footway.demanding_mountain_hiking" = "Path";
 
-"type.highway.footway.difficult_alpine_hiking" = "Camino";
+"type.highway.footway.difficult_alpine_hiking" = "Path";
 
 "type.highway.footway.hiking" = "Sendero";
 
-"type.highway.footway.mountain_hiking" = "Camino";
+"type.highway.footway.mountain_hiking" = "Path";
```

Comment:
I've no idea why these turned back to English but they should still say "Camino"

Reply:
Thanks for this catch, for some reason es-MX sometimes doesn't  generate right inside of WSL but works fine in native Linux @biodranik

Reply:
This is a flaky issue reproduced randomly. It would be great to find why it happens. Does it depend on some locale env var?